test/class_relationships_test.rb

```diff
@@ -1,74 +1,82 @@
 require "test_helper"
 
-[
-  [::Object, ::Module, ::Class],
-  [Toy::Object, Toy::Module, Toy::Class]
-].each do |_Object, _Module, _Class|
-  describe "relationships between Object, Module and Class" do
-    describe "#class" do
-      specify "Object’s class is Class" do
-        assert_equal _Class, _Object.class
-      end
-
-      specify "Module’s class is Class" do
-        assert_equal _Class, _Module.class
-      end
-
-      specify "Class’s class is Class" do
-        assert_equal _Class, _Class.class
-      end
-    end
-
-    describe "#superclass" do
-      specify "Class’s superclass is Module" do
-        assert_equal _Module, _Class.superclass
-      end
-
-      specify "Module’s superclass is Object" do
-        assert_equal _Object, _Module.superclass
-      end
+[::Object, ::Toy].each do |ns|
+  describe "in the #{ns} namespace" do
+    before do
+      @Class = ns::Class
+      @Module = ns::Module
+      @Object = ns::Object
     end
 
-    describe "#kind_of?" do
-      describe "Object" do
-        specify "Object is a kind of Object" do
-          assert_kind_of _Object, _Object
+    describe "relationships between Object, Module and Class" do
+      describe "#class" do
+        specify "Object’s class is Class" do
+          assert_equal @Class, @Object.class
         end
 
-        specify "Object is a kind of Module" do
-          assert_kind_of _Module, _Object
+        specify "Module’s class is Class" do
+          assert_equal @Class, @Class.class
         end
 
-        specify "Object is a kind of Class" do
-          assert_kind_of _Class, _Object
+        specify "Class’s class is Class" do
+          assert_equal @Class, @Class.class
         end
       end
 
-      describe "Module" do
-        specify "Module is a kind of Object" do
-          assert_kind_of _Object, _Module
-        end
-
-        specify "Module is a kind of Module" do
-          assert_kind_of _Module, _Module
+      describe "#superclass" do
+        specify "Class’s superclass is Module" do
+          puts @Module.to_s
+          puts @Class.to_s
+          puts @Class.superclass
```

Comment:
🤔

Reply:
Whoops -- this doesn't belong here 🤦‍♀️  (Cherry-picked this off the branch where I was trying to turn our singletons into `ClassInstances` and this test was failing so I was.. erm print debugging 😂 )